app/src/pages/createhome.tsx

```diff
@@ -0,0 +1,93 @@
+import Button from "@components/button";
+import FieldInput from "@components/fieldinput";
+import type { NextPage } from "next";
+import Head from "next/head";
+import React from "react";
+import { trpc } from "utils/trpc";
+import { useState } from 'react';
+import { useRouter } from 'next/router';
+
+const CreationPage: NextPage = () => {
+    const [error, setError] = useState<string | null>(null);
+    const router = useRouter();
+
+    const addUserToHome = trpc.useMutation(["occupies.addUserToHome"], {
+        onError: (error) => {
+            setError(error.message);
+        },
+    });
+
+    const createHome = trpc.useMutation(["home.createHome"], {
+        onError: (error) => {
+            setError(error.message);
+        },
+        onSuccess: async (home) => {
+            // Add userid and homeid to the occupies table
+            await addUserToHome.mutateAsync({
```

Comment:
Nice, would like to refactor some of the existing code to handle mutations in this way. Error Handling does work